libs/wire-api/src/Wire/API/Conversation/Action.hs

```diff
@@ -89,6 +89,40 @@ instance ToJSON SomeConversationAction where
         actionJSON = fromMaybe A.Null $ schemaOut (conversationActionSchema sb) action
      in A.object ["tag" A..= tag, "action" A..= actionJSON]
 
+instance S.ToSchema SomeConversationAction where
```

Comment:
i guess it's ok to not use schema-profunctor, but then can you add a roundtrip- and/or unit-test?

Reply:
Roundtrip tests for the `ToJSON` and `FromJSON` instances, which I haven't touched, already exist. The `ToSchema` instance is just for the API docs.